package/src/lib.rs

```diff
@@ -110,6 +111,7 @@ pub enum BuildCommand {
         #[clap(long)]
         intermediate: bool,
     },
+    /// List the build commands that will be run
```

Comment:
Is this comment in the right spot? I don't think the `Package` command has changed in this PR

Reply:
No, whoops, not sure how it ended up there.